pkg/nuctl/command/delete.go

```diff
@@ -51,13 +51,13 @@ func newDeleteCommandeer(rootCommandeer *RootCommandeer) *deleteCommandeer {
 
 type deleteFunctionCommandeer struct {
 	*deleteCommandeer
-	functionConfig functionconfig.Config
+	functionConfigs []functionconfig.Config
 }
 
 func newDeleteFunctionCommandeer(deleteCommandeer *deleteCommandeer) *deleteFunctionCommandeer {
 	commandeer := &deleteFunctionCommandeer{
 		deleteCommandeer: deleteCommandeer,
-		functionConfig:   *functionconfig.NewConfig(),
+		functionConfigs:  []functionconfig.Config{},
 	}
 
 	cmd := &cobra.Command{
```

Comment:
The command's `Use` should specify that you can use several function names (same for get)